packages/core/src/base/typography/_typography.docs.scss

```diff
@@ -0,0 +1,106 @@
+/*
+Typography
+
+The design system doesn't style base HTML typography elements (like `h1`, `h2`, `p`, etc) in order to avoid conflicting with any existing site styles. You should instead use one of the base typography classes below to apply styling:
+
+- `ds-display`
+- `ds-title`
+- `ds-h{level}` where `level` is a number between `1` and `6`
+- `ds-text` and `ds-text--lead`
+
+Markup: typography.example.html
+
+Style guide: style.typography
+*/
+
+/*
+Links
+
+Links connect users to a different page or further information.
+
+The only base HTML element the design system applies styling to is the `<a>` element. To prevent this, you can override the `$ds-include-base-html-rulesets` Sass variable and set it to `false`.
+
+Markup: links.example.html
```

Comment:
I wish the markup was inserted where it appears in the file instead of at the bottom, this makes the page not look so great